src/scs/pki.md

```diff
@@ -0,0 +1,31 @@
+# Public Key Infrastructure
+
+In SCS context a public key infrastructure (PKI) term refers to a system for creation, storage, and distribution of digital certificates which are used to verify that a particular key belongs to a certain entity. PKI is creating and managing a hierarchical set of digital certificates which map public keys to certain entities. Storage and revocation methodologies are to be defined.
+
+The PK of SCS should consist of:
+  + A certificate authority (CA) responsible for storing, issuing and signing the digital certificates
+  + A registration authority (RA) responsible for requesting entity identity verification
+  + A central directory responsible for secure storage of the keys
+  + A Certificate Management System (CMS) managing access to stored certificates
+ 
+## Private Certificate Authority (PCA)
+
+PCA enables creation of private certificate authority hierarchies, consisting of root and subordinate CAs. It issues end-entity X.509 certificates, that are used for: 
+
+  + Encrypted TLS communication channels (data encryption in transit)
+  + Code and image signing
+
+PCA can be established in the cloud or on premise. Initially, openssl based on-prem solution is assumed, however, some of the target projects might consider using commercial cloud solutions.
+
+### CA Hierarchy Options
+
+CA usually consists of:
```

Comment:
Same here: for this list remove all 'a' articles in the items. Do not forget to begin each entry in a bulleted list with a capital letter.